packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -5748,7 +5748,7 @@ class _ScribblePlaceholder extends WidgetSpan {
 /// See also:
 ///
 ///  * [String.runes], which deals with code points like this class.
-///  * [String.characters], which deals with graphemes.
+///  * [Characters], which deals with graphemes.
```

Comment:
I tried to docImport the `characters` package but that didn't work for the extension on String:
```
/// @docImport 'package:characters/characters.dart';
```
(defined here https://main-api.flutter.dev/flutter/characters/StringCharacters.html)